README.md

```diff
@@ -41,45 +46,74 @@ Before you begin, make sure you have the following installed on your system:
     git clone https://github.com/citrineos/citrineos-core
     ```
 
-1. Navigate to the CitrineOS Server directory:
+1. Install project dependencies from root dir:
 
-    ```shell
-    cd citrineos-core/Server
-    ```
+   ```shell
+   npm run install-all
+   ```
 
-1. Install project dependencies:
+1. Build project from root dir:
 
    ```shell
-   ./unix-init-install-all.sh
+   npm run build
    ```
 
-1. Start the server and its supporting infrastructure with:
+1. The docker container should be initialized by running `docker-compose -f ./docker-compose.yml up -d` or
+   by using the IntelliJ `Server` Run Configuration which was created for this purpose.
```

Comment:
The instructions should `cd` into `Server/` and just run `docker compose up`. Also, we should not not be prescriptive about IDEs. The references to IntelliJ need to be removed. It would be easier to understand the changes in this file if they weren't combined with changes to the formatting.

Reply:
We should talk about having instructions for most common IDEs

Reply:
adjusted readme to mention `Server/` dir

Reply:
Noted! [I've opened a Github discussion.](https://github.com/orgs/citrineos/discussions/18)